frontend/src/scenes/billing/v2/PlanTable.tsx

```diff
@@ -418,47 +173,37 @@ export function PlanTable({ redirectPath }: { redirectPath: string }): JSX.Eleme
-                        <tr
-                            key={product}
-                            className={
-                                i !== Object.keys(billingPlans[0].products).length - 1 ? 'PlanTable__tr__border' : ''
-                            }
-                        >
-                            <th scope="row">
-                                {billingPlans[0].products[product].name}
-                                <p className="ml-0 text-xs text-muted mt-1">{billingPlans[0].products[product].note}</p>
-                            </th>
-                            {billingPlans.map((plan) => (
-                                <td key={`${plan.name}-${product}`}>
-                                    {plan.products[i].tiers.map((tier) => (
-                                        <div
-                                            key={`${plan.name}-${product}-${tier.description}`}
-                                            className="flex justify-between items-center"
-                                        >
-                                            <span className="text-xs">{tier.description}</span>
-                                            <span className="font-bold">{tier.price}</span>
-                                        </div>
-                                    ))}
-                                    {plan.name !== 'PostHog Cloud Lite' ? (
-                                        <Link
-                                            to="https://posthog.com/pricing"
-                                            target="_blank"
-                                            className="text-xs font-semibold"
-                                        >
-                                            More volume tiers
-                                        </Link>
-                                    ) : null}
-                                </td>
-                            ))}
-                        </tr>
-                    ))}
+                    {billing?.available_plans
+                        ? billing?.available_plans[0].products
+                              .filter((product) => product.type !== 'base' && product.type !== 'enterprise')
+                              .map((product, i) => (
+                                  <tr
+                                      key={product.type}
+                                      className={
+                                          billing?.available_plans?.[0].products.length &&
+                                          i !== billing?.available_plans?.[0].products.length - 1
+                                              ? 'PlanTable__tr__border'
+                                              : ''
+                                      }
+                                  >
+                                      <th scope="row">
+                                          {product.type === 'events' ? 'Product analytics' : 'Session recording'}
```

Comment:
We can retrieve the unit from Stripe, for the future

Reply:
Done.